ptf/sai_base_test.py

```diff
@@ -276,7 +276,7 @@ def check_cpu_port_hdl(self):
                 index=True,
                 parent_scheduler_node=True)
             self.assertEqual(queue, q_attr['index'])
-            self.assertEqual(self.cpu_port_hdl, q_attr['port'])
+            self.assertEqual(self.cpu_port_hdl, q_attr['port']-1)
```

Comment:
Please add the enhancement points to the changes in the code, it can help understand the reason for the changes.
Please avoid unnecessary changes.